index.js

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env node
-if (process.env.DISABLE_OPENCOLLECTIVE && process.env.DISABLE_OPENCOLLECTIVE !== '0' && process.env.DISABLE_OPENCOLLECTIVE.toLowerCase() !== 'false') {
-  return;
-}
-var pkg = require(require('path').resolve('./package.json'));
-if (pkg.collective) {
-  console.log(`\u001b[96m\u001b[1mThank you for using ${pkg.name}!\u001b[96m\u001b[1m`);
-  console.log(`\u001b[0m\u001b[96mIf you rely on this package, please consider supporting our open collective:\u001b[22m\u001b[39m`);
-  console.log(`> \u001b[94m${pkg.collective.url}/donate\u001b[0m\n`);
-}
+var envDisable = Boolean(process.env.DISABLE_OPENCOLLECTIVE);
```

Comment:
Looks cleaner, but that doesn't cover `DISABLE_OPENCOLLECTIVE=false` which will be true.

Boolean('false') -> true